src/tools/spake2p/README.md

```diff
@@ -31,3 +31,12 @@ random Salts and corresponding Verifiers):
 ```
 ./spake2p gen-verifier --count 100 --iteration-count 15000 --salt-len 32 --out spake2p-provisioning-data.csv
 ```
+
+Example command that generates 100 sets of spake2p parameters (Specific PIN
+Codes, random Salts and corresponding Verifiers):
+
```

Comment:
please describe the content of `pincodes.txt`

Reply:
Done